WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -60,6 +60,8 @@ final class PostListViewController: AbstractPostListViewController, UIViewContro
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        wpAssertionFailure("test", userInfo: nil)
+
```

Comment:
Looks like this was left over from testing

Reply:
Fixed